Applications/Spire/Include/Spire/SpireTester/ListModelTester.hpp

```diff
@@ -5,6 +5,8 @@
 #include <doctest/doctest.h>
 #include "Spire/Spire/ListModel.hpp"
 
+using namespace boost;
```

Comment:
Don't add ```using namespace``` in header files.

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-using-directive